### DISCUSSION TITLE
Search for Service Mapping

### DISCUSSION THREAD

**Discussion by matt-edu** [netbox-community/netbox#5566] [Q&A] [answered]
Is there any way to make Service Mappings show up in a global search?

We use Netbox to model/document several F5 load balancers, each with over 100 VIPs listening on different ports.  Each VIP/port combination is listed under Service Mappings with a unique name that matches the VS name on the load balancer itself.

Our intent is to use netbox to quickly determine which load balancer a particular service is hosted on, but these service names are not showing up in a global search so it's like we haven't entered them at all.

**Comment by ryanmerolle** [comment, marked as answer]
You can do this only via https://demo.netbox.dev/ipam/services/ search screen and not global

https://github.com/netbox-community/netbox/issues/4643
